### ISSUE TITLE
take a look at the tail end of the observed graph below

### ISSUE THREAD

**Issue by mcolvin** [mcolvin/Bluff-Lake-Project#20]
@VictoriaStarnes The decline in lake WSE makes me think we are missing some water loss. Could it be attributed to having gates open?
The model does pretty darn good for the summer months, which is the most important as water is really limited then... So that is good.

![image](https://user-images.githubusercontent.com/654215/94620172-c7b5b100-0273-11eb-93f9-ffdeea773693.png)

**Comment by VictoriaStarnes** [comment]
They shouldn't have opened the gates. Plus, I don't have any notes that it was open while I was at the refuge. 

They could have flooded some of the back fields or GTRs. I can check the water management plan before our meeting tomorrow. Plus, I have a Friends of Noxubee meeting on Thursday so I should have an opportunity to corner Travis and ask him about it if we still have questions.

Also, there's a strange dip in the observed WSE in mid-May. Is that just because the graph is compressed?

**Comment by mcolvin** [comment]
I'm not sure about the dip in mid-may it might be that I average Cypress and the gauge loggers to get the overall lake elevation take a look into it and see if that might be the case. I also have a email into Steve to see if maybe evaporation is a bigger factor in the overall water budget than we might expect.

**Comment by VictoriaStarnes** [comment]
There is a dip in the intake WSE for May. In the raw pressure data drops in the middle of a data file, so it's not a conversion/input error.
![image](https://user-images.githubusercontent.com/45180904/94634190-f98b3f80-0294-11eb-9156-9602488aa921.png)

I also noticed that there was a small error in the load-and-clean code for adding model_data$wse_lake. It is now fixed and the updated code is pushed.
